github/table_github_repository_collaborator.go

```diff
@@ -66,6 +95,9 @@ func tableGitHubRepositoryCollaboratorList(ctx context.Context, d *plugin.QueryD
 		_, err := plugin.RetryHydrate(ctx, d, h, listPage, retryConfig())
 		plugin.Logger(ctx).Debug(rateLimitLogString("github_repository_collaborator", &query.RateLimit))
 		if err != nil {
+			if strings.Contains(err.Error(), "You do not have permission to view repository collaborators") {
```

Comment:
Do we need this separate log statement if we have the `Error` log statement below, which will show up in the plugin logs?